projects/rero/ng-core/src/lib/record/record-ui.service.ts

```diff
@@ -143,6 +143,15 @@ export class RecordUiService {
   canUpdateRecord$(record: object, type: string): Observable<ActionStatus> {
     const config = this.getResourceConfig(type);
 
+    if (config.permissions) {
+      const permissions = config.permissions(record);
+      if ('canUpdate' in permissions) {
+        return permissions.canUpdate.pipe(
```

Comment:
`permissions.canUpdate` returns an observable containing: 
```
{
  can: true,
  message: ''
}
```

I think pipe/map should not be applied here as we have already the value of `permission.canUpdate`.

The same comment is valid for the other permissions.

Reply:
done